src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DefaultCircleRadius = 10;
+    private static final int LimitValue = 20;
+    private static final int AmountOfFigures = 5;

```

Comment:
check constant naming
```suggestion
    private static final int DefaultCircleRadius = 10;
    private static final int LimitValue = 20;
    private static final int AmountOfFigures = 5;

```